int/api/server.go

```diff
@@ -91,20 +92,23 @@ func NewServer(flags StartServerFlags) *Server {
 	}
 
 	localAPI := operations.NewMassastationAPI(swaggerSpec)
-	server := restapi.NewServer(localAPI)
+	apiServer := restapi.NewServer(localAPI)
 
-	setAPIFlags(server, flags)
+	setAPIFlags(apiServer, flags)
+	apiServer.GracefulTimeout = 5 * time.Second
 
 	err = store.NewStore()
 	if err != nil {
 		logger.Fatal(err.Error())
 	}
 
-	return &Server{
-		api:      server,
+	server := &Server{
+		api:      apiServer,
 		localAPI: localAPI,
 		shutdown: make(chan struct{}),
 	}
+
+	return server
```

Comment:
this change is unecessary, you can directly return the Server object.

and so the renaming `apiServer := restapi.NewServer(localAPI)` is unnecessary too.